sql/plsql/PlSqlLexer.g4

```diff
@@ -2225,6 +2226,7 @@ LAG:                          'LAG';
 LEAD:                         'LEAD';
 MAX:                          'MAX';
 MEDIAN:                       'MEDIAN';
+MEMOPTIMIZE:                  'MEMOPTIMIZE';
```

Comment:
I guess it also should be added to `non_reserved_keywords_in_12c` or to `non_reserved_keywords_pre12c`.